cpp/ql/test/library-tests/dataflow/taint-tests/test_default_diff.ql

```diff
@@ -0,0 +1,95 @@
+import cpp
+import semmle.code.cpp.dataflow.TaintTracking
+import semmle.code.cpp.security.Security
+import semmle.code.cpp.ir.dataflow.DefaultTaintTracking as IRDefaultTaintTracking
+import semmle.code.cpp.dataflow.DataFlow as ASTDataFlow
+import semmle.code.cpp.ir.dataflow.DataFlow as IRDataFlow
+
+// Only perform taint tracking from parameters, global variables or field variables
+private class TestSecurityOptions extends SecurityOptions {
+  override predicate isUserInput(Expr expr, string cause) {
+    (
+      exists(Parameter p | p.getAnAccess() = expr) or
+      expr.(VariableAccess).getTarget().getParentScope() instanceof GlobalNamespace or
+      exists(Field f | f.getAnAccess() = expr)
+    ) and
+    cause = ""
+  }
+}
+
+// Mimic the definitions of sources and sinks in DefaultTaintTracking
+class DefaultTaintTrackingLookalikeConfig extends TaintTracking::Configuration {
+  DefaultTaintTrackingLookalikeConfig() { this = "DefaultTaintTrackingLookalikeConfig" }
```

Comment:
The reference to compare to should be `semmle.code.cpp.security.TaintTracking`, not `semmle.code.cpp.dataflow.TaintTracking`. The former library is already configured to have the right sources. This change should also allow you to get rid of `equivalent`.

Reply:
You're correct! Fixing the imports removed the need for `equivalent`.